inst/bib/ecolmodell2020.bib

```diff
@@ -26495,3 +26496,1736 @@ @Article{Ghani:2023:MDW
+}
+
+@Article{Rai:2023:MGR,
+  author =       "Anu Rai and Tawqir Bashir and Elio Guarionex
+                 {Lagunes--D{\'\i}az} and Bibek Shrestha",
+  title =        "Modeling \geoname{Ganges} river dolphin distribution
+                 and prioritizing areas for efficient conservation
+                 planning --- a range-wide assessment",
+  journal =      j-ECOL-MODELL,
+  volume =       "481",
+  number =       "??",
+  pages =        "??--??",
+  month =        jul,
+  year =         "2023",
+  CODEN =        "ECMODT",
+  DOI =          "https://doi.org/10.1016/j.ecolmodel.2023.110362",
+  ISSN =         "0304-3800 (print), 1872-7026 (electronic)",
+  ISSN-L =       "0304-3800",
+  bibdate =      "Mon Apr 24 08:20:07 MDT 2023",
+  bibsource =    "http://www.math.utah.edu/pub/tex/bib/ecolmodell2020.bib",
+  URL =          "http://www.sciencedirect.com/science/article/pii/S030438002300090X",
+  acknowledgement = ack-nhfb,
+  ajournal =     "Ecol. Modell.",
+  articleno =    "110362",
+  fjournal =     "Ecological Modelling: International Journal on
+                 Ecological Modelling and Systems Ecology",
+  journal-URL =  "https://www.journals.elsevier.com/ecological-modelling",
+}
+
+@Article{Mugabi:2023:EDC,
+  author =       "Francis Mugabi and Kevin J. Duffy",
+  title =        "Epidemiological drivers and control strategies for
+                 \geoname{African} swine fever transmission cycles at a
+                 wildlife-livestock interface",
+  journal =      j-ECOL-MODELL,
+  volume =       "481",
+  number =       "??",
+  pages =        "??--??",
+  month =        jul,
+  year =         "2023",
+  CODEN =        "ECMODT",
+  DOI =          "https://doi.org/10.1016/j.ecolmodel.2023.110344",
+  ISSN =         "0304-3800 (print), 1872-7026 (electronic)",
+  ISSN-L =       "0304-3800",
+  bibdate =      "Mon Apr 24 08:20:07 MDT 2023",
+  bibsource =    "http://www.math.utah.edu/pub/tex/bib/ecolmodell2020.bib",
+  URL =          "http://www.sciencedirect.com/science/article/pii/S0304380023000728",
+  acknowledgement = ack-nhfb,
+  ajournal =     "Ecol. Modell.",
+  articleno =    "110344",
+  fjournal =     "Ecological Modelling: International Journal on
+                 Ecological Modelling and Systems Ecology",
+  journal-URL =  "https://www.journals.elsevier.com/ecological-modelling",
+}
+
+@Article{Ramampiandra:2023:CML,
+  author =       "Emma Chollet Ramampiandra and Andreas Scheidegger and
+                 Jonas Wydler and Nele Schuwirth",
+  title =        "A comparison of machine learning and statistical
+                 species distribution models: Quantifying overfitting
+                 supports model interpretation",
+  journal =      j-ECOL-MODELL,
+  volume =       "481",
+  number =       "??",
+  pages =        "??--??",
+  month =        jul,
+  year =         "2023",
+  CODEN =        "ECMODT",
+  DOI =          "https://doi.org/10.1016/j.ecolmodel.2023.110353",
```

Comment:
@seananderson and @ericward-NOAA this new article might be of interest to both of you